lib/creator/yeoman/utils/entry.js

```diff
@@ -8,7 +8,7 @@ module.exports = (self, answer) => {
 		result = self.prompt([
 			InputValidate(
 				'multipleEntries',
-				'Type the name you want for your modules (entry files), separated by comma [example: \'app\']',
+				'Type the names you want for your modules (entry files), separated by comma [example: \'app,vendor\']',
```

Comment:
Surely the example should match the question. The question clearly wants **multiple** entry files, **comma-separated**, so the example should match.

If I have misinterpreted this I suggest the question needs re-wording.

Not sure if we're having a language issue here, perhaps input from @TheLarkInn would help.